tests/test_standard_model.py

```diff
@@ -123,103 +123,103 @@ def random_model(random_data_df_dict):
     return model
 
 
-# def test_populate_model(data_df_dict):
```

Comment:
Can't believe I didn't notice this in #57, but I commented some tests out while debugging other tests and forgot to come back and uncomment 😬